.github/workflows/upload_new_asset.yml

```diff
@@ -19,12 +19,12 @@ jobs:
           fetch-depth: 2
```

Comment:
In the secrets section above, you'll need to add the env var like:

```
INVESTIGATORS_GH_TO_NAME_MAP: ${{ secrets.INVESTIGATORS_GH_TO_NAME_MAP }}
```

Reply:
done in last commit